front/src/assets/compositionCode/stdcmCompositionCodes.ts

```diff
@@ -0,0 +1,21 @@
+const COMPOSITION_CODES = [
```

Comment:
Ah, one more thing: do we really want to put this file in `assets/`? Assets are copied over in a directory when building the web app, but this is just a .ts file imported like any other.

Reply:
It's true that he could be somewhere else. In `src/application/stdcm/const.ts` perhaps?

Reply:
Yeah, somewhere in `src/application/stdcm/` sounds good to me.

Reply:
Even in `const` ?

Reply:
Either `src/application/stdcm/consts.ts` (we use the plural form everywhere else), or `src/application/stdcm/consts/compositionCodes.ts` if it's deemed too much data (with the upcoming rolling stock list).

Reply:
Done